src/resource/Payment.php

```diff
@@ -0,0 +1,604 @@
+
+        return $this;
+    }
+
+    /**
+     * Set installment count.
+     *
+     * @param int $installmentCount
+     *
+     * @return $this
+     */
+    public function setInstallmentCount($installmentCount)
+    {
+        $this->data->installmentCount = $installmentCount;
+
+        return $this;
+    }
+
+    /**
+     * Set statement descriptor.
+     *
+     * @param string $statementDescriptor
+     *
+     * @return $this
+     */
+    public function setStatementDescriptor($statementDescriptor)
+    {
+        $this->data->statementDescriptor = $statementDescriptor;
+
+        return $this;
+    }
+
+    /**
+     * Set payment means made available by banks.
+     *
+     * @param string           $bankNumber     Bank number. Possible values: 001, 237, 341, 041.
+     * @param \DateTime|string $expirationDate Date of expiration debit.
+     * @param string           $returnUri      Return Uri.
+     *
+     * @return $this
+     */
+    public function setOnlineBankDebit($bankNumber, $expirationDate, $returnUri)
+    {
+        if ($expirationDate instanceof \DateTime) {
+            $expirationDate = $expirationDate->format('Y-m-d');
+        }
+        $this->data->fundingInstrument->method = self::METHOD_ONLINE_BANK_DEBIT;
+        $this->data->fundingInstrument->onlineBankDebit = new stdClass();
+        $this->data->fundingInstrument->onlineBankDebit->bankNumber = $bankNumber;
+        $this->data->fundingInstrument->onlineBankDebit->expirationDate = $expirationDate;
+        $this->data->fundingInstrument->onlineBankDebit->returnUri = $returnUri;
+
+        return $this;
+    }
+
+    /**
+     * Set Multiorders.
+     *
+     * @param \Moip\Resource\Multiorders $multiorder
+     *
+     * @return $this
+     */
+    public function setMultiorder(Multiorders $multiorder)
+    {
+        $this->multiorder = $multiorder;
+
+        return $this;
+    }
+
+    /**
+     * Set order.
+     *
+     * @param \Moip\Resource\Orders $order
+     *
+     * @return $this
+     */
+    public function setOrder(Orders $order)
+    {
+        $this->order = $order;
+
+        return $this;
+    }
+
+    /**
+     * Turns on a delay on credit card payment capture (pre-authorization).
+     *
+     * @return $this
+     */
+    public function setDelayCapture()
+    {
+        $this->data->delayCapture = true;
+
+        return $this;
+    }
+    
+    /**
+     * Set escrow to a payment
+     *
+     * @param string $description
+     * 
+     * @return $this
+     */
+    public function setEscrow($description) 
+    {
+        $this->data->escrow = new stdClass();
+        $this->data->escrow->description = $description;
+
+        return $this;
+    }
+
+    /**
+     * Capture a pre-authorized amount on a credit card payment.
+     *
+     * @throws \Exception
+     *
+     * @return Payment
+     */
+    public function capture()
+    {
+        if ($this->order !== null) {
```

Comment:
Sugestão: 

`if (is_null($this->order)) {
        throw new \Exception('Sorry, multipayment capture is not available on this version');
} 
$path = sprintf('/%s/%s/%s/%s', MoipResource::VERSION, self::PATH, $this->getId(), 'capture');`